pkg/server/routes.go

```diff
@@ -131,13 +154,13 @@ func (s *Server) routes(
 
 	serviceConfig := services.NewConfig(s.logger, ldClient)
 
-	store, err := storage.NewStore(
+	store, storeErr = storage.NewStore(
```

Comment:
I'm not sure I understand why we'd need to re-init `store` here, if it's already been created above